app/questionnaire/questionnaire_schema.py

```diff
@@ -467,6 +495,9 @@ def get_summary_options(self) -> ImmutableDict[str, bool]:
     def get_sections(self) -> Iterable[ImmutableDict]:
         return self._sections_by_id.values()
 
+    def get_section_ids(self) -> Iterable[str]:
```

Comment:
return type should be concrete, so `list[str]`? (same above)

Reply:
Fails the mypy check as list as the return type is `dict_keys`

Reply:
ah ok - all good then